tests/test_fields.py

```diff
@@ -238,3 +238,35 @@ class Config:
     assert str(PreferenceSchema2.__fields__["preferred_food"].type_) != str(
         PreferenceSchema.__fields__["preferred_food"].type_
     )
+
+
+@pytest.mark.django_db
+def test_listing():
+    class ListingSchema(ModelSchema):
+        class Config:
+            model = Listing
+            use_enum_values = True
+
+    import json
```

Comment:
Please remove this import and the debug print.

Reply:
:+1: